05/app.js

```diff
@@ -0,0 +1,113 @@
+class Student {
+  constructor (name, surname) {
+    this.firstName = name,
+    this.surName = surname,
+    this.grades = {}
+  }
+
+  addGrade(subject, grade) {
+    const lowerCaseSubject = subject.toLowerCase();
+    if (!this.grades[lowerCaseSubject]) {
+      const subjects = [
```

Comment:
Co jeśli pojawi się nowy przedmiot? Raczej starałbym się nie ograniczać listy w tym miejscu. 
Zrobiłbym to poza klasą `Student`, aby dodanie nowego przedmiotu nie zmuszała mnie do zmiany tej klasy, które nie odpowiada za ten element. To tak na przyszłość - rozumiem, że takie było zadanie więc nie ma co wychodzić poza niego ;)